v2/internal/controllers/samples_test.go

```diff
@@ -27,6 +27,7 @@ const samplesPath = "../../config/samples"
 var skipTests = []string{
 	// TODO: Will re-record test .. as resource was having some issues in creation.
 	"cache",
+	"subscription", // Can't be run in our live env currently
```

Comment:
Maybe you want to delete this? We already have a skip flag for live

Reply:
We can't easily record it, is the issue actually. I need to fix the comment to explain better. Since we can't record it there is no recording so it wont' work.

Reply:
Ive updated the comment here